apps/cowswap-frontend/src/modules/volumeFee/state/safeAppFeeAtom.ts

```diff
@@ -68,5 +66,11 @@ export const safeAppFeeAtom = atom<VolumeFee | null>((get) => {
     return isStableCoinTrade ? FEE_PERCENTAGE_BPS.STABLE.TIER_3 : FEE_PERCENTAGE_BPS.REGULAR.TIER_3
   })()
 
+  console.debug('[Volume Fee] Calculated a fee for Safe App (not added yet)', {
+    bps,
+    fiatAmount,
+    isStableCoinTrade,
+  })
```

Comment:
were you intending to add this message?

Reply:
Yes. It's not easy for Elena to test the volume fee calculation, so I added this debug log.